utils/utils-lib/src/constants.rs

```diff
@@ -45,6 +45,9 @@ pub const CREATED_BY_KEY: &str = "openebs.io/created-by";
 /// The value to mark the creation source of a pool to be disk pool operator in labels.
 pub const DSP_OPERATOR: &str = "operator-diskpool";
 
+/// The value to mark the creation source of a pool to be msp operator in labels.
+pub const MSP_OPERATOR: &str = "msp-operator";
```

Comment:
maybe keep this in the migration module, since we don't need it otherwise.

Reply:
Done.